resources/authgear/templates/en/web/verify_magic_link.html

```diff
@@ -43,7 +50,13 @@ <h1 class="m-0 primary-txt text-center text-xl font-bold">{{ template "verify-ma
 	{{ end }}
 
 {{ else }}
-<div class="text-sm break-words primary-txt text-center">{{ template "verify-magic-link-matched-description" }}</div>
+{{ if $.ClientName }}
+	<div class="mb-3 text-sm break-words primary-txt text-center whitespace-pre-wrap">{{ template "verify-magic-link-matched-description" (dict "AppOrClientName" $.ClientName) }}</div>
+	{{ else }}
+	{{ $appName := ($.Translations.RenderText "app.name" nil) }}
+	<div class="mb-3 text-sm break-words primary-txt text-center whitespace-pre-wrap">{{ template "verify-magic-link-matched-description" (dict "AppOrClientName" $appName) }}</div>
+	{{ end }}
+</div>
```

Comment:
1. The indentation is wrong.
2. Unexpected `</div>` tag?
3. In the email, we are using the app name. Keep it simple, maybe we also always use the app name here?